src/components/notifications/PageNotifications.vue

```diff
@@ -12,37 +12,45 @@
       <div slot="subtitle">Don't worry, they are on their way!</div>
     </TmDataMsg>
 
-    <router-link
-      v-for="notification in notifications"
-      :key="notification.id"
-      class="notification"
-      :to="notification.link"
+    <EventList
+      v-else
+      :events="notifications"
+      :more-available="moreAvailable"
+      @loadMore="loadMore"
     >
-      <div class="content">
-        <img :src="notification.icon" />
-        <div>
-          <h3 class="title">{{ notification.title }}</h3>
-        </div>
-      </div>
-      <i class="material-icons notranslate">chevron_right</i>
-    </router-link>
+      <template scope="event">
```

Comment:
```suggestion
      <template slot-scope="event">
```

Fixes the "slot-scope" warning. Also needed in PageBlock and PageTransactions

Reply:
Ok, I will push this better